tools/pontoon/team_project.md

```diff
@@ -0,0 +1,68 @@
+
+This is a screenshot of the [Italian team page](https://pontoon.mozilla.org/it).
+
+![Team page for Italian](/assets/images/pontoon/team_project/team_page.png)
+
+The page header contains linguistic information about this locale (plural form, writing direction, etc.), plus an indication of the overall status of completion and statistics.
+
+![Header of team page for Italian](/assets/images/pontoon/team_project/team_page_header.png)
+
+The [Teams page](https://pontoon.mozilla.org/teams/) will have similar information, but referring to Pontoon as a whole. For example: number of teams, team with most translations or missing strings, overall completion and statistics across projects and locales.
+
+There are 4 tabs available (highlighted in red), 5 if you’re an *admin* or *manager*.
+
+* **Projects**: a list of all projects enabled in Pontoon for this locale.
+* **Contributors**: a list of active contributors with their statistics, filterable by time (all time, 1/3/6/12 months).
+* **Bugs**: a list of open bugs for this locale, retrieved from [Bugzilla](https://bugzilla.mozilla.org/).
+* **Info**: information about the team. This is currently only editable by administrators ([bug 1343804](https://bugzilla.mozilla.org/show_bug.cgi?id=1343804)).
+* **Permissions**: manage user permissions (see [this document](users.md#managing-permissions) for details).
+
+Hovering a project in the list will replace the progress bar with a set of detailed statistics (untranslated strings, missing strings, etc.). Note that all these numbers are links, so you can use them to open the project with a filter already enabled, for example to display only missing strings. You can click on the number of total strings to access *All Resources* (i.e. all files) for this project.
+
+![Project hover](/assets/images/pontoon/team_project/project_hover.png)
+
+## Project page
+
+The layout of the project page is very similar to the team page, displaying a list of available files instead of projects.
+
+![Header of project page](/assets/images/pontoon/team_project/project_page_header.png)
+
+The page header contains useful information about this project:
+* **Priority**: it goes from 5 stars (highest priority) to 1 star (lowest priority).
+* **Deadline**: set only for some projects, it indicates when the translation is due.
+* **Repository**: link to the repository storing translations for this project.
+* **Resources**: links to useful external resources. Custom links can be defined for each project.
+* **Contact person**: localization driver in charge of this project.
+
+There are 4 tabs available (highlighted in red):
+* **Resources**: files available in the project.
+* **Contributors**: a list of active contributors with their statistics, filterable by time. Unlike the tab available in the team page, this only includes contributions to this specific project.
```

Comment:
contributions to this **localization** (of a particular project to a particular locale).